pkg/api/resource/quantity.go

```diff
@@ -112,10 +112,19 @@ func MustParse(str string) Quantity {
 	return *q
 }
 
+// Scale is used for getting and setting the scaled value.
+type Scale int
+
+const (
+	Nano  Scale = -9
+	Micro Scale = -6
+	Milli Scale = -3
```

Comment:
I was trying to understand the reason these were defined from the comments,
and couldn't.  Once you go > 0, you have to address why you have base-10s
but not base-2 suffixes

On Tue, Dec 1, 2015 at 5:24 PM, Tim St. Clair <notifications@github.com>
wrote:

> In pkg/api/resource/quantity.go
> <https://github.com/kubernetes/kubernetes/pull/17548#discussion_r46366531>
> :
>
> > @@ -112,10 +112,19 @@ func MustParse(str string) Quantity {
> >  	return *q
> >  }
> >
> > +// Scale is used for getting and setting the scaled value.
> > +type Scale int
> > +
> > +const (
> > +	Nano  Scale = -9
> > +	Micro Scale = -6
> > +	Milli Scale = -3
>
> Correct about base-10 (will update Scale comment). As for larger values,
> it's an arbitrary decision but asking for smaller values seemed like the
> common case. I'll add up to Exa since thats what we have in suffixes.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/17548/files#r46366531>.
>